DXMainClient/DXGUI/Multiplayer/GameLobby/GameLobbyDropDown.cs

```diff
@@ -104,13 +118,12 @@ static string Localize(XNAControl control, string attributeName, string defaultV
         /// <summary>
         /// Applies the drop down's associated code to spawn.ini.
         /// </summary>
-        /// <param name="spawnIni">The spawn INI file.</param>
         public void ApplySpawnIniCode(IniFile spawnIni)
         {
-            if (dataWriteMode == DropDownDataWriteMode.MAPCODE || SelectedIndex < 0 || SelectedIndex >= Items.Count)
+            if (SelectedIndex < 0 || SelectedIndex >= Items.Count)
```

Comment:
Are you sure to remove this check, instead of checking `dataWriteMode == DropDownDataWriteMode.MAPCODE || dataWriteMode == DropDownDataWriteMode.SPAWN_SPAWNMAP`? I haven't looked into details so no matter either choice is selected, could you provide some explanations on this line?